### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by ivuk**
Just a few typo fixes. Also adds the `cowsay` module mentioned in the paragraph describing the example copy command (it was missing for some reason).

Hope you're having a great day! :)

### CODE REVIEW

File: en_us/quests/control_repository.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Copy the site-specific `cowsay`, `pasture`, `motd`, `user_accounts`, `role`,
 and `profile` Puppet modules into the your control repository's `site`
 directory.
 
-    cp -r {pasture,motd,user_accounts,role,profile} /root/control-repo/site/
+    cp -r {cowsay,pasture,motd,user_accounts,role,profile} /root/control-repo/site/
```
Comment (greglarkin) [inline en_us/quests/control_repository.md]: Would you mind making this same change in the `ja_jp/quests` version of the file too?

**Comment by greglarkin** [comment]
Thank you @ivuk , and I added one follow-up question for you.

File: en_us/quests/control_repository.md (hunk 1 above)
Comment (ivuk) [reply en_us/quests/control_repository.md]: Oh, I missed that one, let me check it out. Thanks! :)

Comment (ivuk) [reply en_us/quests/control_repository.md]: @greglarkin, as far as I can see, that command is also mentioned in tests/control_repository_spec.rb, should I update it there as well?

Comment (greglarkin) [reply en_us/quests/control_repository.md]: Yes, it looks correct to me to add that directory to that test and also check for its existence just below the copy command.

Comment (ivuk) [reply en_us/quests/control_repository.md]: Thanks, I'll push that change as well.

Comment (ivuk) [reply en_us/quests/control_repository.md]: Pushed, looks OK to me.

Comment (greglarkin) [reply en_us/quests/control_repository.md]: 👍

**Comment by greglarkin** [comment]
Thank you @ivuk !

**Pull request merged**